### PR TITLE
make_ui

### PR DESCRIPTION
**Pull request by koval1off**
create_mail
create user
make table with data about mails

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -4,23 +4,50 @@
...
         mail_from = mail[3]
         direction = "<<--" if mail_from == client_id else "-->>"
         read_status = "[v]" if mail[2] else "[0]"
-        print(f"{direction} ID: {mail[0]} Body: {mail[1]} {read_status}")
-      
+        data = [direction, mail[0], mail[1], read_status]
+        output_data.append(data)
+
+    data = pandas.DataFrame(output_data, columns=headers)
+    print(data.to_string(index=False))
```
Comment (koval1off) [inline main.py:49]: image

Comment (koval1off) [reply main.py:49]: now it looks like this. 
I convert all to string to get rid of indices at the left side of the table

Comment (koval1off) [reply main.py:49]: image

Comment (koval1off) [reply main.py:49]: here is the result without this line
I think you don't really need this explanation, but I'm training to explain my ideas and my decisions

**Review by Alex-Krykun — APPROVED** [review]

**Pull request merged**